setup.py

```diff
@@ -40,9 +40,21 @@
         "sourced-ml-core>=0.0.5",
         "PyStemmer>=1.3,<2.0",
         "humanize>=0.5.0,<0.6",
+        "clickhouse-driver>=0.1.1,<1.0",
+        "modelforge>=0.14.1,<1.0",
+        "scipy>=1.0,<2.0",
+        "jinja2>=2.10.1<3.0",
     ],
     tests_require=["docker>=3.6.0,<4.0"],
-    package_data={"": ["LICENSE.md", "README.md"]},
+    package_data={
+        "": ["LICENSE.md", "README.md"],
+        "sourced": [
+            os.path.join(d, fp).replace("sourced/", "")
```

Comment:
Now that we dropped 3.5, we can forget about these `os.path`-s and use much better `pathlib` API everywhere.

```python
from pathlib import Path

pkg_path = Path("sourced/ml/mining")

SourceFileLoader("...", pkg_path / "__init__.py")

(Path(__file__).parent / "README.md").open(encoding="utf-8")

pkg_path.rglob("*.json*") + pkg_path.rglob("*.jinja2")
```

Reply:
nice !